src/config.rs

```diff
@@ -235,6 +235,10 @@ pub struct Config {
     #[serde(default)]
     colors: ColorList,
 
+    /// Cursor type
+    #[serde(default)]
+    cursor_style: CursorStyle,
```

Comment:
It would be nice if this just stored an `ansi::CursorStyle` so that usage of the `config::CursorStyle` type is limited to this module.